dynamic-tools/zap-extensions/wso2-jira-issue-creator/wso2jiraissuecreator/ExtensionJiraIssueCreater.java

```diff
@@ -0,0 +1,268 @@
+/*
+ *
+ *  * Copyright (c) 2014, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Since this is a new file that you write, the copyright year should be current year